include/swift/Basic/Statistic.h

```diff
@@ -13,7 +13,10 @@
 #ifndef SWIFT_BASIC_STATISTIC_H
 #define SWIFT_BASIC_STATISTIC_H
 
+#include "llvm/ADT/SmallString.h"
 #include "llvm/ADT/Statistic.h"
+#include "swift/Basic/Timer.h"
+#include "swift/FrontendTool/FrontendTool.h"
```

Comment:
Tricky. How best to make this interface accept callbacks of the appropriate type, then? Or should I move this class out of Basic and into something higher level (eg. FrontendTool itself? Is it appropriate for Driver to see that?)